Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -1226,6 +1226,10 @@ bool WebProcessProxy::dispatchSyncMessage(IPC::Connection& connection, IPC::Deco
 
 void WebProcessProxy::didClose(IPC::Connection& connection)
 {
+#if ENABLE(CLOSE_WEBCONTENT_XPC_CONNECTION_POST_LAUNCH)
```

Comment:
I'm a little confused why we need this and I'm worried this may cause us to ignore certain crashes.
This is supposed to get called when the IPC connection gets closed. Our IPC connection uses mach and not xpc, therefore, why does it get called if we kill the xpc_connection but do not terminate the child process? Can't we avoid this getting called at all in this case?

What if this is a real child process crash that happened on launch and we haven't closed the xpc connection yet?

Reply:
Good point! It turns out this method is not called with the current location of the `xpc_connection_cancel` call, so I have removed it in the latest version.

Thanks for reviewing!